app/helpers/users_helper.rb

```diff
@@ -1,8 +1,8 @@
 module UsersHelper
-  # Returns the Gravatar for the given user.
-  def gravatar_for(user, size: 80)
+  def gravatar_for(user, options = {size: 80}) # Returns the Gravatar for the given user.
```

Comment:
if you have a hash, you need to write it that way `options = { size: 80 }` (with whitespaces on both sides)